package.json

```diff
@@ -15,7 +15,7 @@
   "devDependencies": {},
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "nodemon src/server.ts"
+    "start": "nodemon src/Routes/routes.ts"
```

Comment:
we shouldn't make this change.
the server configuration should live in `server.ts`, only the routes to be moved to a separate file